### PR TITLE
Record start of run

### PR DESCRIPTION
**Pull request by bricehalder**

### CODE REVIEW

File: apis/runs.py [hunk 1]
```diff
@@ -18,8 +18,8 @@ def create_run():
 
     # datetime wants timestamp in seconds since epoch
     start_time = datetime.fromtimestamp(request.json['start_time']/1000)
-    end_time = datetime.fromtimestamp(request.json['end_time']/1000)
-    path = json.dumps(request.json['path'])
+    end_time = datetime.fromtimestamp(request.json['end_time']/1000) if 'end_time' in request.json else None
```
[the remark this answers is not in the document]
Comment (dqian3) [reply apis/runs.py:21]: Also this would create 2 runs, would it not? Shouldn't we have an update somewhere?

**Review by dqian3 — CHANGES_REQUESTED** [review]
See the comment about the

File: apis/runs.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bricehalder) [reply apis/runs.py:21]: Fixed

**Comment by bricehalder** [comment]
Need to do `ALTER TABLE runs MODIFY path TEXT NULL;` on prod database.

Took out path for same reason as end time

**Comment by dqian3** [comment]
> Need to do `ALTER TABLE runs MODIFY path TEXT NULL;` on prod database.
> 
> Took out path for same reason as end time

Shouldn't this be changed in the scripts then?

**Review by dqian3 — APPROVED** [review]

**Pull request merged**